ui/src/scss/_vanilla-overrides.scss

```diff
@@ -4,17 +4,6 @@
  * {Link to relevant GitHub issue}
  */
 
-@import "~vanilla-framework/scss/settings_colors";
-@import "~vanilla-framework/scss/settings_spacing";
-@import "~vanilla-framework/scss/patterns_icons";
-@import "~vanilla-framework/scss/global_functions";
-
-// 19-08-2019 Caleb: Form submit button too close to input when $multi = 1
-// https://github.com/canonical-web-and-design/vanilla-framework/issues/2497
-.p-form__group:last-child {
-  margin-bottom: $spv-outer--medium;
```

Comment:
As noted in the QA notes maybe this one can't be removed yet?

Reply:
Upgrading to Vanilla 2.8.0 meant this rule broke a few other things. :( I've left it removed but I've added another issue for a slightly different regression.